config/routes.rb

```diff
@@ -1,26 +1,30 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Information for the application
   get 'info',  to: 'information#info'
   get 'help',  to: 'information#help'
   get 'about', to: 'information#about'
 
-  # Temporary root
-  root to: "information#about"
+  root to: "application#login"
```

Comment:
Devise need a root to: that actually points where you want to go after log in, it will change your options to login when not authenticated